site/content/docs/user/using-wsl2.md

```diff
@@ -58,10 +58,16 @@ wsl --set-default-version 2
 1. Now, you can install your Linux distro of choice by searching the Windows Store. If you don't want to use the Windows Store, then follow the steps in the WSL docs for [manual install](https://docs.microsoft.com/en-us/windows/wsl/install-manual).
 1. Start up your distro with the shortcut added to the start menu
 
-## Setting up Docker in WSL2
+## Setting up Docker in WSL2 with Docker Desktop
 
 Install Docker with WSL2 backend here: https://docs.docker.com/docker-for-windows/wsl/
 
+
+## Setting up Docker in WSL2 without Docker Desktop
+
+An alternative approach that does not require the use of the heavy Docker Desktop. The installation procedure can be found at https://dev.to/bowmanjd/install-docker-on-windows-wsl-without-docker-desktop-34m9
```

Comment:
I'm not sure we should make judgements on the tool like "heavy", the kind project isn't favoring one over the other. (We *do* currently favor _Docker_ at all over podman because podman is less stable - as in it makes breaking major version changes etc - and as a result our support is as well)

Reply:
I actually was comparing Docker Desktop vs Engine when I meant heavy. Do you want me to rephrase or remove `too heavy ` for clarity?

Reply:
let's drop the heavy comment, it's not necessary. users can determine which works best for them, we're just noting kind will work with these.

Reply:
```suggestion
Alternatively, docker can be installed in WSL2 without using Docker Desktop
See for example: https://dev.to/bowmanjd/install-docker-on-windows-wsl-without-docker-desktop-34m9
```

Though, this guide may be a confusing example as it also suggests installing podman, I'm not sure podman will work with kind in WSL2.

Reply:
Do you want me to add few more posts or find a newer better one? The posts work though if we neglect the podman section